app/components/work_packages/info_line_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<%=
+  flex_layout do |flex|
+    flex.with_column(mr: 2) do
+      render(WorkPackages::HighlightedTypeComponent.new(work_package: @work_package, font_size: :small))
+    end
+    flex.with_column(mr: 2)  do
+      render(Primer::Beta::Text.new(font_size: :small, color: :muted)) { "##{@work_package.id}" }
+    end
+    flex.with_column do
+      render WorkPackages::StatusBadgeComponent.new(status: @work_package.status)
+    end
```

Comment:
Currently, the font-size is always set to `small` and the ID always `muted`. I guess, it makes sense to make at least the font_size configurable. The default could still be `small`.